src/plugins/english/webnovel.ts

```diff
@@ -0,0 +1,313 @@
+/* eslint-disable no-case-declarations */
```

Comment:
what is this for

Reply:
Copied over from my template. I removed it.